BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpViewController.swift

```diff
@@ -0,0 +1,170 @@
+        super.viewDidLoad()
+
+        setupUI()
+    }
+    
+    private func setupUI() {
+        configureBackground()
+        
+        addWelcomeLabel()
+        addAppNameLabel()
+        addLogoImageView()
+        addPhraseLabel()
+        addStartButton()
+        
+        setupViewConstraints()
+    }
+    
+    private func configureBackground() {
+        view.backgroundColor = UIColor(hexNumber: 0xCFCFCF)
+        
+        view.addSubview(backgroundImageView)
+        backgroundImageView.image = UIImage(named: "mountains")
+    }
+    
+    private func addWelcomeLabel() {
+        view.addSubview(welcomeLabel)
+        
+        welcomeLabel.text = String(localized: "welcomeLabelText").uppercased()
+        welcomeLabel.font = UIFont(name: FontName.SFProDisplayMedium.rawValue, size: 16.0)
+        welcomeLabel.textAlignment = .center
+        welcomeLabel.textColor = .black
+    }
+    
+    private func addAppNameLabel() {
+        view.addSubview(appNameLabel)
+
+        appNameLabel.text = "HABIT HERO".uppercased()
+        appNameLabel.font = UIFont(name: FontName.SFProDisplayBold.rawValue, size: 24.0)
+        appNameLabel.textAlignment = .center
+        appNameLabel.textColor = .black
+    }
+    
+    private func addLogoImageView() {
+        view.addSubview(logoImageView)
+        addLogoLabel()
+        
+        logoImageView.image = UIImage(named: "logo")
+    }
+    
+    private func addLogoLabel() {
+        logoImageView.addSubview(logoLabel)
+        
+        logoLabel.text = String(localized: "logoLabelText").uppercased()
+        logoLabel.font = UIFont(name: FontName.interExtraBold.rawValue, size: 15.0)
+        logoLabel.textAlignment = .left
+        logoLabel.textColor = UIColor(hexNumber: 0xFFFFFF)
+    }
+    
+    private func addPhraseLabel() {
+        view.addSubview(phraseLabel)
+        
+        phraseLabel.text = String(localized: "phraseLabelText").uppercased()
+        phraseLabel.font = UIFont(name: FontName.SFProDisplayMedium.rawValue, size: 14.0)
+        phraseLabel.textAlignment = .center
+        phraseLabel.textColor = .black
+        phraseLabel.lineBreakMode = .byWordWrapping
+        phraseLabel.numberOfLines = 0
+    }
+    
+    private func addStartButton() {
+        view.addSubview(startButton)
+        
+        var buttonConfiguration = UIButton.Configuration.filled()
+        buttonConfiguration.title = String(localized: "startButtonTitle").uppercased()
+        buttonConfiguration.cornerStyle = .large
+        buttonConfiguration.baseBackgroundColor = UIColor(hexNumber: 0x090909)
+
+        startButton.configuration = buttonConfiguration
+        
+        startButton.addTarget(self, action: #selector(startButtonTapped), for: .touchUpInside)
+    }
+    
+    private func setupViewConstraints() {
+        let safeArea = view.safeAreaLayoutGuide
+        
+        backgroundImageView.snp.makeConstraints { make in
+            let heightMultiplier: CGFloat = 445 / 844
```

Comment:
445 / 844
what this numbers mean?

Reply:
It's the ratio of the screen height to background image view height on the mockup. This multiplier is used to calculate the backgroundImageView height.

Reply:
That's will not work due to different sizes of the screen

Reply:
So here you want to make image height equals to half of screen height?

Reply:
In general - yes, but slightly more than half. In this case, can this restriction be applied?

Reply:
Define multiplier as a constant, not equation

Reply:
heightMultiplier was removed, multiplier value was added instead